server/admin_management/api/app/routers/router_access_control_privilege.py

```diff
@@ -32,7 +27,7 @@
 
 @router.post(
     "",
-    response_model=List[schemas.FamAccessControlPrivilegeCreateResponse],
+    response_model=schemas.FamAccessControlPrivilegeResponse,
```

Comment:
Hi Ian, I think you already imported the FamAccessControlPrivilegeResponse from schemas, so maybe no need to call it schemas.FamAccessControlPrivilegeResponse

Reply:
Right, maybe I was confused that time from IDE, will remove it.